lib/extras/dec/apng.cc

```diff
@@ -745,7 +745,6 @@ Status DecodeImageAPNG(const Span<const uint8_t> bytes,
     pframe.frame_info.duration = frame.duration;
     pframe.frame_info.layer_info.blend_info.blendmode =
         should_blend ? JXL_BLEND_BLEND : JXL_BLEND_REPLACE;
-    pframe.frame_info.layer_info.have_crop = 1;
```

Comment:
Shouldn't this be true in case x0/y0 are nonzero or xsize/ysize do not match the image dimensions?

Reply:
Yes, added that.